doc/start/client.rst

```diff
@@ -0,0 +1,30 @@
+=======================
+Basic Ceph Client Setup
+=======================
+Client machines need some basic configuration in order to interact with
+a cluster. This document describes how to configure a client machine
+for cluster interaction.
+
+Note that most client machines only need the `ceph-common` package and
+its dependencies installed.
+
+Config File Setup
+=================
+Client machines can generally get away with a smaller config file than
+a full-fledged cluster member. To generate a minimal config file, log
+into a machine that is already a cluster member and run::
+
+    ceph config generate-minimal-conf
+
+This will generate a minimal config file that is mostly just for telling
+the client how to reach the Ceph Monitors. The file should typically be
+installed in `/etc/ceph/ceph.conf`.
+
+Keyring Setup
+=============
+Most Ceph clusters are run with encryption enabled, and the client will
```

Comment:
encryption is not supported so far. I think you mean authentication

Reply:
Yes, thanks. Will fix.